README.md

```diff
@@ -746,7 +759,12 @@ Ph.D. Thesis, ETH-Zurich, Zurich, Switzerland, 2017. [PDF](https://www.research-
 *[Event-based Vision for High-Speed Robotics](http://rpg.ifi.uzh.ch/docs/PhD17_Mueggler.pdf)*,  
 Ph.D. Thesis, University of Zurich, Zurich, Switzerland, 2017.
 - See also [Theses from Delbruck's group at INI](https://www.ini.uzh.ch/~tobi/wiki/doku.php?id=publications#phd_thesis)
-
+- <a name="Matolin10PhD"></a>Matolin, D.,
```

Comment:
Please reformat the entry (separate line for author, title and institution), to comply with the rest of the entries.
An additional title in English is recommended.

Reply:
Also, put the entry in its right position chronologically.

Reply:
Ok